pkg/router/controller/router_controller.go

```diff
@@ -227,6 +228,70 @@ func (c *RouterController) HandleEndpoints(eventType watch.EventType, obj interf
 	c.Commit()
 }
 
+// HandleEndpointSlice handles a single EndpointSlice event and refreshes the router backend.
+func (c *RouterController) HandleEndpointSlice(eventType watch.EventType, objMeta metav1.ObjectMeta, items []discoveryv1beta1.EndpointSlice) {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	ipAddress := func(addressType discoveryv1beta1.AddressType, address string) string {
+		switch addressType {
+		case discoveryv1beta1.AddressTypeIPv4, discoveryv1beta1.AddressTypeIPv6:
+			return address
+		case discoveryv1beta1.AddressTypeFQDN:
+			return address // TODO(frobware): resolve?
+		default:
```

Comment:
Should the default case print a log (or error)?

Reply:
Removed this function - we don't need it anymore.